mood-analysis.rb

```diff
@@ -28,5 +74,9 @@ def analyze_mood(words)
   "05/11 Yay, yay, yay! I'm having a awfuly great day."
 ]
 
-puts analyze_mood(text[0])
-puts analyze_mood(text[1])
+text.each do |aline|
+  line = strip_punctuation(aline)
+  face = analyze_mood(line)
```

Comment:
Is there any case where we would NOT want to strip_punctuation before analyze_mood? If there isn't, the we should just call strip_punctuation inside analyze_mood